src/browser/queries/wdio-types.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="@testplane/webdriverio" />
+/*
+eslint-disable @typescript-eslint/no-explicit-any
+*/
+
+export type SelectorsBase = {
+    $: WebdriverIO.Browser["$"];
+    $$: WebdriverIO.Browser["$$"];
+};
+
+export type BaseWithExecute = {
```

Comment:
Why are these needed at all? We already have typings for execute, addCommand, etc everywhere.